application/config.json.template

```diff
@@ -89,8 +89,7 @@
     "logInfoChannelWebhook": "<put_your_webhook_here>",
     "logErrorChannelWebhook": "<put_your_webhook_here>"
     "starboard": {
-        "oofEmojiName": "oof"
-        "lmaoEmojiName": "lmao"
-        "starboardChannelId": <put_channel_id_here>
+        "starboardEmojiNames" : ["oof", "lmao"]
+        "starboardChannelName": "oofs_and_lmaos"
```

Comment:
the prefixes on on the names are not necessary due to the `starboard: ...` already grouping them